phalcon/mvc/view/engine/volt/compiler.zep

```diff
@@ -1779,10 +1779,10 @@ class Compiler implements InjectionAwareInterface
 		 * Echo statement
 		 */
 		if this->_autoescape {
-			return "<?php echo $this->escaper->escapeHtml(" . exprCode . "); ?>";
+			return "<?= $this->escaper->escapeHtml(" . exprCode . ") ?>";
 		}
 
-		return "<?php echo " . exprCode . "; ?>";
+		return "<?= " . exprCode . " ?>";
```

Comment:
Existence or absence of a semicolon doesn't influence behavior in this case. Moreover, in official documentation such option of syntax is given in such look: http://php.net/manual/en/function.echo.php